### ISSUE TITLE
Crash when selecting imported game while offline

### ISSUE THREAD

**Issue by billybruiser** [JosefNemec/Playnite#237]
[playnite crash.zip](https://github.com/JosefNemec/Playnite/files/1411981/playnite.crash.zip)

If you select a game that has been added to Playnite, but hasn't populated the images yet, while your internet connection is disabled, Playnite crashes with the attached error.

**Comment by inondle** [comment]
Looks like a pretty straightforward null ref exception. If @JosefNemec  doesn't mind I could take a crack at patching it.

**Comment by JosefNemec** [comment]
Released in 2.9

**Comment by sigmap** [comment]
Issue is still present in 2.9 when trying to access steam games.
```
2017-12-10 12:02:38.5668|ERROR:Failed to download metadata for id:900, provider:Steam.System.Net.WebException: The remote server returned an error: (404) Not Found.
   at System.Net.WebClient.DownloadDataInternal(Uri address, WebRequest& request)
   at System.Net.WebClient.DownloadData(Uri address)
   at System.Net.WebClient.DownloadData(String address)
   at Playnite.Providers.Steam.SteamLibrary.DownloadGameMetadata(Int32 id)
   at Playnite.Providers.Steam.SteamLibrary.UpdateGameWithMetadata(IGame game)
   at Playnite.Database.GameDatabase.UpdateGameWithMetadata(IGame game)
   at PlayniteUI.MainWindow.DownloadMetadata(GameDatabase database, Provider provider, ProgressControl progresser, CancellationToken token)
2017-12-10 12:09:02.8507|ERROR:Unhandled exception occured.System.Net.WebException: The remote name could not be resolved: 'steamcdn-a.akamaihd.net'
   at System.Net.WebClient.DownloadFile(Uri address, String fileName)
   at System.Net.WebClient.DownloadFile(String address, String fileName)
   at Playnite.Web.DownloadFile(String url, String path)
   at PlayniteUI.ImageUrlToImageSourceConverter.Convert(Object value, Type targetType, Object parameter, CultureInfo culture)
   at System.Windows.Data.BindingExpression.TransferValue(Object newValue, Boolean isASubPropertyChange)
   at MS.Internal.Data.ClrBindingWorker.CompleteGetValue(AsyncGetValueRequest request)
   at MS.Internal.Data.ClrBindingWorker.OnCompleteGetValueOperation(Object arg)
   at MS.Internal.Data.DataBindOperation.Invoke()
   at MS.Internal.Data.DataBindEngine.ProcessCrossThreadRequests()
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.DispatcherOperation.InvokeImpl()
   at System.Windows.Threading.DispatcherOperation.InvokeInSecurityContext(Object state)
   at MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at MS.Internal.CulturePreservingExecutionContext.Run(CulturePreservingExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Windows.Threading.DispatcherOperation.Invoke()
   at System.Windows.Threading.Dispatcher.ProcessQueue()
   at System.Windows.Threading.Dispatcher.WndProcHook(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
   at MS.Win32.UnsafeNativeMethods.DispatchMessage(MSG& msg)
   at System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)
   at System.Windows.Threading.Dispatcher.PushFrame(DispatcherFrame frame)
   at System.Windows.Application.RunDispatcher(Object ignore)
   at System.Windows.Application.RunInternal(Window window)
   at System.Windows.Application.Run(Window window)
   at PlayniteUI.App.Main()
```

It can be reproduced by disabling network connection.
